src/test_l1_hdmi_cec_driver.c

```diff
@@ -1078,21 +1074,21 @@ void test_hdmicec_hal_l1_getLogicalAddress_sinkDevice_positive ( void )
 
     result = HdmiCecOpen (&handle);
     //if init is failed no need to proceed further
-    UT_ASSERT_EQUAL ( result, HDMI_CEC_IO_SUCCESS );
+    UT_ASSERT_EQUAL_FATAL ( result, HDMI_CEC_IO_SUCCESS );
 
     result = HdmiCecAddLogicalAddress( handle, logicalAddress );
-    if (HDMI_CEC_IO_SUCCESS != result) { HDMICEC_ASSERT_AUTO_TERM(); UT_FAIL ("HdmiCecAddLogicalAddress failed"); }
+    if (HDMI_CEC_IO_SUCCESS != result) { HDMICEC_TERM(); UT_FAIL_FATAL ("HdmiCecAddLogicalAddress failed"); }
 
     result = HdmiCecGetLogicalAddress(handle, &logicalAddressCrossCheck);
-    if (HDMI_CEC_IO_SUCCESS != result) { HDMICEC_ASSERT_AUTO_TERM(); UT_FAIL ("HdmiCecGetLogicalAddress failed"); }
-    UT_ASSERT_TRUE( logicalAddress!= logicalAddressCrossCheck);
+    if (HDMI_CEC_IO_SUCCESS != result) { HDMICEC_TERM(); UT_FAIL_FATAL ("HdmiCecGetLogicalAddress failed"); }
+    UT_ASSERT_TRUE_FATAL( logicalAddress== logicalAddressCrossCheck);
```

Comment:
comparing these two values, the result is pass only when they both are same.. but here it will get aborted.

Reply:
to avoid confuse
if(logicalAddress != logicalAddressCrossCheck){
              UT_FAIL("logicalAddress 0x%x and logicalAddressCrossCheck 0x%x are not same\n",logicalAddress,logicalAddressCrossCheck);
      }